pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -154,11 +154,18 @@ self: super: builtins.intersectAttrs super {
   # Add necessary reference to gtk3 package
   gi-dbusmenugtk3 = addPkgconfigDepend pkgs.gtk3 super.gi-dbusmenugtk3;
 
-  hs-mesos = overrideCabal (drv: {
-    # Pass _only_ mesos; the correct protobuf is propagated.
-    extraLibraries = [ pkgs.mesos ];
-    preConfigure = "sed -i -e /extra-lib-dirs/d -e 's|, /usr/include, /usr/local/include/mesos||' hs-mesos.cabal";
-  }) super.hs-mesos;
```

Comment:
@sternenseemann you forgot to remove https://github.com/NixOS/nixpkgs/blob/6b2d239024f405d60d98d12f8f31dcfd42ee6a94/pkgs/top-level/packages-config.nix#L37